### PR TITLE
LPS-199641

### PR DESCRIPTION
**Pull request by TamyrisTorres**
Ticket:https://liferay.atlassian.net/browse/LPS-199641

### CODE REVIEW

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:hotel

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 1]
```diff
@@ -647,6 +649,8 @@ public class UpgradeCatchAllCheck {
 	@Reference
 	private LayoutService _layoutService;
 
+	private ServiceTrackerList<Class<E>> _serviceTrackerList;
+
```
[the remark this answers is not in the document]
Comment (TamyrisTorres) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:653]: @natocesarrego Yes, It's correct. I double checked and the SF didn't no change.

**Comment by natocesarrego** [comment]
Manually forwarded to Brian: https://github.com/brianchandotcom/liferay-portal/pull/144889

**Pull request closed without merging**